pyvera/__init__.py

```diff
@@ -293,13 +307,7 @@ def set_value(self, name, value):
                         'action': 'Set' + name,
                         'newLoadlevelTarget': value}
                 else:
-                    payload = {
```

Comment:
I think you've moved half the logic to a new function here, which slightly obscures the issue over handling handling the `newLoadlevelTarget` special case. 

The existing code isn't great - certainly worth refactoring and perhaps handling `newLoadlevelTarget` issue as a special case without repeating the whole dictionary construction.

Happy for you to complete the refactor - or just put back (and I'll do it another time) as you wish.